src/algebra/module/linear_map.lean

```diff
@@ -279,6 +279,19 @@ end add_comm_group
 
 end linear_map
 
+namespace module
+
+/-- In the presence of the module `module.comp_hom S g`, `g : R →+* S` is an `R`-linear map. -/
```

Comment:
I don't understand this docstring :confused:

Reply:
As above,
```suggestion
/-- `g : R →+* S` is `R`-linear when the module structure on `S` is `module.comp_hom S g` . -/
```

Reply:
Yup!